app/pages/EditBreakingNewsPage.scss

```diff
@@ -0,0 +1,58 @@
+.edit-breaking-news-header, .edit-breaking-news-footer {
+    display: flex;
+    justify-content: center;
+}
+
+.edit-breaking-news-header {
+    font-size: 28px;
+    font-weight: 600;
+    padding: 10px 0px 0px;
+}
+
+.edit-breaking-news-footer {
+    padding-bottom: 20px;
+}
+
+.news-item-form {
+    margin: 0px 80px;
+    padding: 20px 0px;
+    h2 {
+        font-size: 20px;
+        font-weight: 600;
+    }
+    label {
+        font-size: 18px;
+    }
+}
+
+.news-item-form:not(:last-child) {
+    border-bottom: 1px solid grey;
+}
```

Comment:
I think it might be visually clearer to always insert the bottom border instead of omitting it for the last child, since otherwise there's nothing separating the "Add Breaking News Story" from individual news article form. This could be confusing to users because it would make the "Add Breaking News Story" button look like it'll just save the last news article, but instead it'll actually create a brand new one.

I think part of the disconnect is that most users probably would not expect a blank news article to be created and saved to the database upon clicking the "Add Breaking News Story". Creating an actual divider between the edit forms and the "Add Breaking News Story" button will at least help separate the two, even if it's still potentially misleading to a user.